mbtb_app/web_portal/mbtb_portal/assets/js/suspend_user_with_reason.js

```diff
@@ -0,0 +1,28 @@
+var suspend_user_with_reason = angular.module("suspend_user_with_reason_app", []);
+
+suspend_user_with_reason.controller("suspend_user_with_reason_controller",
+  ['$scope', '$window', '$http', function ($scope, $window, $http) {
+
+    // on-click of confirm button
+    $scope.confirmClick = function () {
+      if (confirm("This action will suspend a user. Are you sure?")) {
+        let url = '/suspend_single_user/';
```

Comment:
9 -> url should be for 2nd controller, change it to: `/suspend_user_with_reason`.